### ISSUE TITLE
Vertical nav doesn't update .active on page change

### ISSUE THREAD

**Issue by tomer** [patternfly/patternfly-react-demo-app#39]
Currently only the **Ipsum** item is `active=true` regardless the active page.

### Steps to reproduce:
* Open the demo app.
* Navigate to a different page than **Ipsum**

### Actual result:
Please notice that the Ipsum menu items keeps highlighted even after navigating away. 

![image](https://user-images.githubusercontent.com/50206/43364445-e9406c7e-9322-11e8-9d79-0249fc4661ce.png)

**Comment by priley86** [comment]
i was noticing the same... will try to resolve this in #36 soon.

**Comment by Mhbuur** [comment]
the current activeItem search implementation ...

```
    const activeItem = this.menu.find(
      item => location.pathname.indexOf(item.to) > -1
    );
```

... will always match the first item '/' for any location. 

Here is a fix for that:

```
    const activeItem = this.menu.find(
      item => location.pathname === item.to
    );
```

**Comment by priley86** [comment]
thanks... apologies, I have not had much time to look at this. I pushed a small change to allow for this in #43 , as well as ensuring the "Ipsum" nav is still selected when subnavs are selected.

**Comment by priley86** [comment]
I should note that, for more complex / auth based routing, we have started the [rcue](https://github.com/patternfly/patternfly-react-demo-app/tree/rcue) branch which uses a custom masthead. May also be useful for some folks.

**Comment by mturley** [comment]
This was resolved by #45.
